pom.xml

```diff
@@ -147,16 +147,16 @@
     <module>bom</module>
     <module>broker</module>
     <module>core</module>
+    <module>service-base</module>
```

Comment:
But why do you need to change the order of the modules in the POM file for that?